src/client.jsx

```diff
@@ -40,14 +40,20 @@ window.errorReporter = ErrorReporter.getInstance({
   componentName,
 });
 
-const app = (
-  <Provider store={store}>
-    <IntlProvider locale={locale.abbreviation} messages={locale.messages}>
-      <BrowserRouter basename={basename}>
-        <App />
-      </BrowserRouter>
-    </IntlProvider>
-  </Provider>
-);
-
-render(app, document.getElementById('root'));
+const renderApp = Component =>
+  render(
+    <Provider store={store}>
+      <IntlProvider locale={locale.abbreviation} messages={locale.messages}>
+        <BrowserRouter basename={basename}>
+          <Component />
+        </BrowserRouter>
+      </IntlProvider>
+    </Provider>,
+    document.getElementById('root'),
+  );
+
+renderApp(App);
+
+if (module.hot) {
```

Comment:
Try something like this:
```
if (module.hot) {
  module.hot.accept('./containers/App/App', () => {
    renderApp();
  });
}
```
To fix redux error

Reply:
@oyvinmar  Prøvde dette først men da funker ikke HMR i det hele tatt. Å kjøre .accept() uten callback var eneste som funket. Du la ikke noe til i createStore?